src/wms/ExportEvents.ts

```diff
@@ -31,5 +33,7 @@ export async function getEvents(exportDate: Date): Promise<FacillitySchedules[]>
     throw error;
   }
 
+  console.info('getEvents ending');
```

Comment:
again maybe successful instead of ending.

Reply:
I'm happy with 'ending' which doesn't indicate any success or failure. The purpose of the log entry is to indicate that the method has run and is about to exit. Handy to determine where any timeouts occur.